data/src/tables/trajectories.rs

```diff
@@ -0,0 +1,36 @@
+use super::*;
+
+use linesonmaps::types::linestringm::LineStringM;
+
+pub struct Trajectories {
+    pub mmsi: Vec<MMSIType>,
+    pub trajectory: Vec<LineStringM>, // Change this to the custom linestringm type.
```

Comment:
```suggestion
    pub trajectory: Vec<LineStringM<4326>>, // Change this to the custom linestringm type.
```
ikke streng nødvendigt